ambry-utils/src/main/java/com.github.ambry.utils/Utils.java

```diff
@@ -51,6 +59,20 @@
    */
   public static final long Infinite_Time = -1;
 
+  /**
+   * Maximum supported time in millis
+   * @return the max supported time in millis
+   */
+  public static long getMaxSupportedTimeInMs() {
+    Date date = null;
+    try {
+      SimpleDateFormat sdf = new SimpleDateFormat(maxAllowableDateForEpochTimeFormat);
+      date = sdf.parse(maxAllowableDateForEpochTime);
+    } catch (ParseException e) {
```

Comment:
we could do that. But felt that we are exposing that we store integers or expiration values at the router layer, which may not be necessary.